pkg/objstore/s3/s3_test.go

```diff
@@ -0,0 +1,51 @@
+package s3
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strings"
+	"testing"
+)
+
+func TestUserAgent(t *testing.T) {
+
+	serverResponse := `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's put it in const ` const serverResponse = ".."` to indicate this variable is immutable.